mgmtfn/dockplugin/dockplugin.go

```diff
@@ -307,10 +319,15 @@ func createEndpoint(tenantName, hostname string) func(http.ResponseWriter, *http
 			return
 		}
 
-		// FIXME:
-		GetEndPointName(cereq.NetworkID, cereq.EndpointID)
+		// Get Service name
+		serviceName, err := GetEndPointName(cereq.NetworkID, cereq.EndpointID)
```

Comment:
I know this is another surface error (sorry!) but `EndPoint` should probably be capitalized along with our convention elsewhere